packages/block-library/src/button/edit.js

```diff
@@ -171,7 +171,7 @@ function ButtonEdit( props ) {
 	return (
 		<>
 			<ColorEdit { ...props } />
-			<Block.div>
+			<div { ...useBlockProps() }>
```

Comment:
For consistency, I think we should avoid inlining any of these hook calls. If an if statement was ever added to implement something like an early return, this would suddenly break the rules of hooks.